### DISCUSSION TITLE
I need help with environment variable.

### DISCUSSION THREAD

**Discussion by cdytoby** [gitextensions/gitextensions#8394] [General] [answered]
I encountered some issue with git and according to a tutorial, I need to do the following for debug purpose:

```
set GIT_TRACE_PACKET=1
set GIT_TRACE=1
set GIT_CURL_VERBOSE=1
```

I did it in bash command, but after I close the command line window and use git extension, it still doesn't give me any more feedback for what's going on.

How do I set these environment variables in gitextensions?

**Comment by pmiossec** [comment, marked as answer]
>I encountered some issue with git and according to a tutorial, I need to do the following for debug purpose:

that's the way to set the env var for the current shell process.

>How do I set these environment variables in gitextensions?

You have to set them at the system level ( https://www.architectryan.com/2018/08/31/how-to-change-environment-variables-on-windows-10/ ) and then restart GitExtensions (sometimes after restarting the system or at least `explorer.exe`)

**Reply by cdytoby** [reply]
Thanks.

**Comment by RussKie** [comment]
You can set an env var in a command prompt, then start GE from this command
prompt, and the app will inherit the setting for the lifetime of the
process.
